docs/source/format/Integration.rst

```diff
@@ -20,8 +20,95 @@
 Integration Testing
 ===================
 
+Our strategy for integration testing between Arrow implementations is:
+
+* Test datasets are specified in a custom human-readable, JSON-based format
+  designed exclusively for Arrow's integration tests
+* Each implementation provides a testing executable capable of converting
+  between the JSON and the binary Arrow file representation
+* The test executable is also capable of validating the contents of a binary
+  file against a corresponding JSON file
+
+Running integration tests
+-------------------------
+
+The integration test data generator and runner uses ``archery``, a Python script
+that requires Python 3.6 or higher. You can create a standalone Python
+distribution and environment for running the tests by using
+`miniconda <https://conda.io/miniconda.html>`_. On Linux this is:
```

Comment:
Do we really have to repeat the Miniconda install instructions on every other documentation page? That isn't scalable. I suggest to either let the user figure it out (it isn't difficult) or to put those in a dedicated page (development FAQ perhaps?).